README.md

```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.org/andela/ah-shakas.svg?branch=develop)](https://travis-ci.org/andela/ah-shakas)
```

Comment:
Can we change this to `master`, I'm gonna explain later on Slack channel why

Reply:
Sure.